se2_planning/src/OmplReedsSheppPlanner.cpp

```diff
@@ -53,6 +53,12 @@ void OmplReedsSheppPlanner::createDefaultStateSpace() {
   setStateSpaceBoundaries();
 }
 bool OmplReedsSheppPlanner::plan() {
+  initialize();
+  ompl::base::StateSpacePtr space = simpleSetup_->getStateSpace();
+  auto bounds = space->as<ompl::base::SE2StateSpace>()->getBounds();
+  //  std::cout << "OmplReedsSheppPlanner: Planner state space bounds: x = [" << bounds.low[0] << ", " << bounds.high[0] << "], y=["
```

Comment:
why do we need lines 56-59?

Reply:
The initialize() call is necessary to set the new state space boundaries in the planner, lines 57-59 were just for debugging to check that the state space is updated correctly.

Reply:
But I think we can just set the new boundaries. There is even a public method for that. We don't need to create the new state space from scratch (which is what these methods will do in the end). Did you ever try that?

Reply:
Adapted in db4a618de5f69d9b9b0bee5375f1217ca36002a6. Is it correct to define the setStateSpaceBoundaries as virtual in the OmplPlanner header file and implement it in the source file? Would you solve it differently?